temporalio/service.py

```diff
@@ -115,6 +115,24 @@ def _to_bridge_config(self) -> temporalio.bridge.client.ClientKeepAliveConfig:
 KeepAliveConfig.default = KeepAliveConfig()
 
 
+@dataclass(frozen=True)
+class HttpConnectProxyConfig:
+    """Configuration for HTTP CONNECT proxy for client connections."""
+
+    target_host: str
+    """Target host:port for the HTTP CONNECT proxy."""
+    basic_auth: Optional[Tuple[str, str]] = None
+    """Basic auth for the HTTP CONNECT proxy if any."""
```

Comment:
Same deal about mentioning tuple is user/pass

Reply:
Fixed